src/Tizen.NUI/src/public/Events/PanGestureDetector.cs

```diff
@@ -341,6 +341,18 @@ public void SetMaximumTouchesRequired(uint maximum)
             if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
         }
 
+        /// <summary>
+        /// This value is a maximum duration of motion can live on the pan gesture event queue.
+        /// If duration exceed it, the motion event is discarded.
+        /// </summary>
+        /// <param name="maximumAgeMilliSecond">Maximum age of motion events as milliseconds</param>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public void SetMaximumMotionEventAge(uint maximumAgeMilliSecond)
```

Comment:
Just my curiosity :
What does 'Age' mean here? `EventAge`

Reply:
Age mean...duration from the event born(?) to now.
This value name comes from UK ; who is native speaker of english.